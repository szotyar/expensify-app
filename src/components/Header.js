import React from 'react';
import {NavLink} from 'react-router-dom';

const Header = () => (
  <header>
    <h1>Expensify</h1>
    <NavLink to="/" >Home</NavLink>
    <NavLink to="/create" >Create</NavLink>
    <NavLink to="/help" >Help</NavLink>
  </header>
);

export default Header;