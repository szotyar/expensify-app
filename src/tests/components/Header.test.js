import { shallow } from 'enzyme';
import React from 'react';
import Header from '../../components/Header';
import toJson from 'enzyme-to-json';

test('I would like it to render header correctly', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
    //expect(wrapper.find('h1').text()).toBe('Expensify');
});