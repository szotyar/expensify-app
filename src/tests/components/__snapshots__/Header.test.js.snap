// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`I would like it to render header correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="active"
    ariaCurrent="true"
    to="/"
  >
    Home
  </NavLink>
  <NavLink
    activeClassName="active"
    ariaCurrent="true"
    to="/create"
  >
    Create Expense
  </NavLink>
</header>
`;
